Packages/vcs/vcs/vector.py

```diff
@@ -59,9 +59,7 @@ def process_src(nm, code):
         sp = scode.split("=")
         nm = sp[0].strip()
         nm = nm.replace("#", "")
-        if nm == "Tl":
```

Comment:
@doutriaux1 Same as before, I deal with 'Tl' later in the code using the new interface.